rotkehlchen/chain/evm/decoding/aura_finance/balances.py

```diff
@@ -0,0 +1,107 @@
+                (HistoryEventType.DEPOSIT, HistoryEventSubType.DEPOSIT_ASSET),
+            },
+        )
+
+    def query_balances(self) -> BalancesSheetType:
+        balances: BalancesSheetType = defaultdict(BalanceSheet)
+        addresses_with_activities = self.addresses_with_activity(event_types={
+            (HistoryEventType.RECEIVE, HistoryEventSubType.RECEIVE_WRAPPED),
+        })
+        pools: set[EvmToken] = set()
+        if len(addresses_with_deposits := list(addresses_with_activities.keys())) == 0:
+            return balances
+
+        for events in addresses_with_activities.values():
+            pools.update(event.asset.resolve_to_evm_token() for event in events)
+
+        chunk_size, call_order = get_chunk_size_call_order(self.evm_inquirer)
+        for pool_token in pools:
+            try:
+                pool_contract = EvmContract(
+                    address=pool_token.evm_address,
+                    abi=self.evm_inquirer.contracts.abi('ERC20_TOKEN'),
+                    deployed_block=0,
+                )
+                results = self.evm_inquirer.multicall(
+                    calls=[
+                        (
+                            pool_token.evm_address,
+                            pool_contract.encode(
+                                method_name='balanceOf',
+                                arguments=[address],
+                            ),
+                        ) for address in addresses_with_deposits
+                    ],
+                    call_order=call_order,
+                    calls_chunk_size=chunk_size,
+                )
+                if not results:
+                    log.error(f'Failed to query aura finance balances for pool {pool_contract.address}')  # noqa: E501
+                    continue
+
+                for idx, result in enumerate(results):
+                    user_address = addresses_with_deposits[idx]
+                    if (balance := pool_contract.decode(
+                        result=result,
+                        method_name='balanceOf',
+                        arguments=[user_address],
+                    )[0]) == 0:
+                        continue
+
+                    formatted_balance = token_normalized_value_decimals(
+                        token_amount=balance,
+                        token_decimals=DEFAULT_TOKEN_DECIMALS,
+                    )
+                    price = get_aura_pool_price(
+                        token=pool_token,
+                        inquirer=Inquirer(),
+                    )
+                    balances[user_address].assets[pool_token] += Balance(
+                        amount=formatted_balance,
+                        usd_value=formatted_balance * price,
+                    )
+            except (RemoteError, IndexError, ValueError) as e:
+                log.error(
+                    f'Failed to query {CPT_AURA_FINANCE} balances for pool {pool_token.evm_address} on '  # noqa: E501
```

Comment:
no need to have a constant as part of a variable string

```suggestion
                    f'Failed to query aura balances for pool {pool_token.evm_address} on '  # noqa: E501
```